src/components/MarketplaceItem.vue

```diff
@@ -24,22 +24,25 @@
     <div class="marketplace-item-footer">
       <button
         v-on:click="goToProject(item)"
-        v-bind:data-tracking-label="item.name"
         v-bind:href="item.sourceUrl"
         class="btn marketplace-item-content__action"
       >
-        Take me to the Project
+        Learn more
```

Comment:
The old market place had "learn more" without capital L.
Do we want to keep that string? 
.. just a question

Reply:
We've agreed on this before.